ark/jsonschema/json.ts

```diff
@@ -0,0 +1,91 @@
+import type { JsonSchemaOrBoolean } from "@ark/schema"
+import { printable, throwParseError } from "@ark/util"
+import { type, type JsonSchema } from "arktype"
+import { parseJsonSchemaAnyKeywords } from "./any.ts"
+import { validateJsonSchemaArray } from "./array.ts"
+import { parseJsonSchemaCompositionKeywords } from "./composition.ts"
+import { validateJsonSchemaNumber } from "./number.ts"
+import { validateJsonSchemaObject } from "./object.ts"
+import { JsonSchemaScope } from "./scope.ts"
+import { validateJsonSchemaString } from "./string.ts"
+
+export const innerParseJsonSchema = JsonSchemaScope.Schema.pipe(
+	(jsonSchema: JsonSchemaOrBoolean): type.Any => {
+		if (typeof jsonSchema === "boolean") {
+			if (jsonSchema) return JsonSchemaScope.Json
+			else return type("never") // No runtime value ever passes validation for JSON schema of 'false'
```

Comment:
You can use `type.never` in cases like this for builtins.

Doesn't make a huge difference, but avoids reparsing for internal stuff.

Reply:
I don't think I realised this syntax existed for builtins; that's quite handy! Updated to use within c7058125